build/steps-build-android.yml

```diff
@@ -31,6 +31,11 @@ steps:
 
 - template: templates/flutter-install.yml
 
+- template : templates/replace-firebase-config.yml
+  parameters:
+    pathToSrc: '${{ parameters.pathToSrc }}'
+    isP
```

Comment:
@Marc-Antoine-Soucy I think there is a typo here, is it normal?

Reply:
I don't think so

Reply:
It just looks like temporary blindness to me on my part

Reply:
Ok I'll fix it.